mu/interface.py

```diff
@@ -755,6 +760,30 @@ def __init__(self, port, theme='day', parent=None):
             raise IOError("Cannot connect to device on port {}".format(port))
         self.set_theme(theme)
 
+    def repl_paste(self):
+        """
+        Grabs clipboard contents then sends down the serial port
```

Comment:
Apologies for I'm being really pedantic here, but it'd be nice to have these comments end with a perior `.`